src/cpp/client.cpp

```diff
@@ -682,12 +676,12 @@ parsed_reply_nested_map Client::get_db_node_info(std::string address)
     if (host.empty() or port == 0)
         throw std::runtime_error(std::string(address) +
                                  "is not a valid database node address.");
-    Command cmd;
+    DBInfoCommand cmd;
     cmd.set_exec_address_port(host, port);
     cmd.add_field("INFO");
     cmd.add_field("everything");
     CommandReply reply = this->_run(cmd);
-    return CommandReplyParser::parse_db_node_info(std::string(reply.str(),
+    return cmd.parse_db_node_info(std::string(reply.str(),
```

Comment:
Since this is a static member function, should we call it like other static functions: ``DBInfoCommand::parse_db_node_info(reply)``?